text/0026-rest-api/rest-api.yaml

```diff
@@ -0,0 +1,895 @@
+      operationId: get-is-confirmed
+      parameters:
+        - in: query
+          name: hashes
+          schema:
+            type: array
+            items:
+              type: string
+          required: true
+          description: Hash of the message.
+      responses:
+        "200":
+          description: successful operation
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/GetIsConfirmedResponse"
+              examples:
+                Example 1:
+                  $ref: '#/components/examples/post-is-referenced-by-milestone-response-example-1'
+        "400":
+          description: unsuccessful operation
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+              examples:
+                default:
+                  $ref: '#/components/examples/invalid-data-provided-response-example'
+        "404":
+          description: unsuccessful operation
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+              examples:
+                default:
+                  $ref: '#/components/examples/can-not-find-data-response-example'
+  /transaction-messages/by-address:
+    get:
+      tags:
+        - transaction-messages
+      summary: Find all transaction-messages that use a given address.
+      description: Find all transaction-messages that use a given address.
+      operationId: get-messages-by-address
+      parameters:
+        - in: query
+          name: hashes
+          schema:
+            type: array
+            items:
+              type: string
+          required: true
+          description: Address of the message.
+      responses:
+        "200":
+          description: successful operation
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/GetMessagesByAddressResponse"
+              examples:
+                default:
+                  $ref: '#/components/examples/post-messages-by-address-response-example-1'
+        "400":
+          description: unsuccessful operation
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+              examples:
+                default:
+                  $ref: '#/components/examples/invalid-data-provided-response-example'
+        "404":
+          description: unsuccessful operation
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+              examples:
+                default:
+                  $ref: '#/components/examples/can-not-find-data-response-example'
+  /outputs/by-hash:
+    get:
+      tags:
+        - transaction-messages
+      summary: Find an output by its hash.
```

Comment:
what is the output hash?

Reply:
This should also be renamed to something like `outputs/by-id` because an output is definitely not identifier via a hash but rather the concatenation of transaction_hash+output_index